org.eclipse.dawnsci.plotting.api/src/org/eclipse/dawnsci/plotting/api/IPlottingSystemViewer.java

```diff
@@ -479,6 +485,11 @@ public ITrace createTrace(String name, Class<? extends ITrace> clazz) {
 			
 			return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1168)